### ISSUE TITLE
.inclusivelintignore is not being considered in the process

### ISSUE THREAD

**Issue by squassina** [inclusivelint/inclusivelint#30]
check 

```bash
function create_ignore_words_list() {
    while read line; do
        IGNORE_DICTIONARY[${line}]=1
    done < <( cat ${BASE_PATH}/.inclusivelint/.inclusivelintignore )
}

function create_dictionary() {
    while read line; do
        key="$(cut -d'=' -f1 <<<"$line")"
        value="$(cut -d'=' -f2 <<<"$line")"
        if [[ ! $key == *" "* ]]; then
            if [[ ! ${IGNORE_DICTIONARY[$key]} ]]; then WARNING_TERMINOLOGIES[$key]="The term $key was detected, consider other terms such as $value instead."; fi
        fi
    done < <( cat ${BASE_PATH}/.inclusivelint/outputRelation.txt )
}
```

**Comment by squassina** [comment]
Assigned to myself.
